components/blocks/newsFeed.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import GhostContentAPI from "@tryghost/content-api";
+import NewsEntry from "./newsEntry";
+
+const api = new GhostContentAPI({
+  url: "https://streamlit.ghost.io",
+  key: "b48c60c1a281bf21dc9afa9950",
```

Comment:
~~@snehankekre is it safe to expose this, or should we use an environmental variable?~~ Just read this:

> These keys are safe for use in browsers and other insecure environments, as they only ever provide access to public data.

So all good! 👍